styles/styles.css

```diff
@@ -0,0 +1,195 @@
+.header {
+  background-color:rgb(9,60,96);
+  width: 100%;
+  color: rgb(9,60,96);
+}
+
+.header-img{
+  width: 33%;
+  padding-top: 3rem;
+  padding-left: 3rem;
+}
+
+h2{
+  color: rgb(9,60,96);
+}
+
+.pricing{
```

Comment:
make sure you have the space after the selector on all elements and after the colon

```
.pricing {
  font-weight: bold;
}
```